examples/plot_diabetes_variable_importance_example.py

```diff
@@ -71,30 +74,12 @@
 # diabetes dataset.
 
 n_folds = 5
-regressor = RidgeCV(alphas=np.logspace(-3, 3, 10))
-regressor_list = [clone(regressor) for _ in range(n_folds)]
-kf = KFold(n_splits=n_folds, shuffle=True, random_state=0)
-for i, (train_index, test_index) in enumerate(kf.split(X)):
-    regressor_list[i].fit(X[train_index], y[train_index])
-    score = r2_score(
-        y_true=y[test_index], y_pred=regressor_list[i].predict(X[test_index])
-    )
-    mse = root_mean_squared_error(
-        y_true=y[test_index], y_pred=regressor_list[i].predict(X[test_index])
-    )
-
-    print(f"Fold {i}: {score}")
-    print(f"Fold {i}: {mse}")
-#############################################################################
-# Fit a baselien model on the diabetes dataset
-# --------------------------------------------
-# We use a Ridge regression model with a 10-fold cross-validation to fit the
-# diabetes dataset.
-
-n_folds = 10
-regressor = RidgeCV(alphas=np.logspace(-3, 3, 10))
+regressor = RidgeCV(
+    alphas=np.logspace(-3, 3, 10),
+    cv=KFold(shuffle=True, random_state=seeds[0]),
```

Comment:
why not simply setting `random_state=1` ?

Reply:
I did it to avoid setting the same seed multiple times. It's for avoiding some indirect correlation due to the random generator.

Reply:
The correlation is there in any case ? Just it's a different one.

Reply:
I prefer to remove numerical artefacts if it's possible. It can help to find bugs if there is one and avoid being hidden due to the usage of the same random generator.

Reply:
You're not removing numerical artefacts here, unless I'm missing something.

Reply:
I don't like to use 2 times the same seed in the same program except when I want the same result. 
However, you seems to really want it, I will do it.